docs/getting-started-guides/mesos-docker.md

```diff
@@ -198,9 +210,9 @@ host machine (mac).
     If you delete the `MESOS_RESOURCES` environment variables, the resource amounts will be auto-detected based on the host resources, which will over-provision by > 2x.
 
     If the configured resources are not available on the host, you may want to increase the resources available to Docker Engine.
-    You may have to increase you VM disk, memory, or cpu allocation in VirtualBox,
-    [Docker Machine](https://docs.docker.com/machine/#oracle-virtualbox), or
-    [Boot2Docker](https://ryanfb.github.io/etc/2015/01/28/increasing_boot2docker_allocations_on_os_x.html).
+    You may have to increase you VM disk, memory, or cpu allocation
+    [Docker Machine Virtualbox](https://docs.docker.com/machine/drivers/virtualbox)
+    [Docker Machine VMware](https://docs.docker.com/machine/drivers/vm-fusion/)
```

Comment:
You may have to increase the Docker Machine VM's disk, memory, or cpu allocation. See the Docker Machine docs for details ([VirtualBox](https://docs.docker.com/machine/drivers/virtualbox), [VMware](https://docs.docker.com/machine/drivers/vm-fusion/)).